x-pack/plugins/alerting/public/hooks/use_create_maintenance_window.ts

```diff
@@ -41,10 +55,11 @@ export function useCreateMaintenanceWindow(props?: UseCreateMaintenanceWindowPro
       );
     },
     onError: (error: IHttpFetchError<KibanaServerError>) => {
+      const getDefaultErrorMessage = (message?: string): string =>
+        message == null || isEmpty(message) ? onErrorWithoutMessage : onErrorWithMessage(message);
```

Comment:
might be simpler to just do `!message ? onErrorWithoutMessage : onErrorWithMessage` 

since empty strings coerces to false, also message can't be null anyways since `message?: string`